plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -93,179 +102,328 @@ function executeCLI($scriptName, &$args) {
 			exit();
 		}
 
-		$data = file($filename);
+		$data = file($filename, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
+		str_getcsv(array_shift($data));
```

Comment:
The `str_getcsv` call does nothing; just use `array_shift` and add a comment to explain:
```php
array_shift($data); // Remove column headers in first row
```